app/components/base/list-and-show-layout/list-and-show-layout.component.ts

```diff
@@ -75,7 +75,7 @@ export class ListAndShowLayoutComponent implements AfterViewInit {
             if (query === "") {
                 this.quickFilter = FilterBuilder.none();
             } else {
-                this.quickFilter = FilterBuilder.prop(this.quickSearchField).startswith(query);
+                this.quickFilter = FilterBuilder.prop(this.quickSearchField).startswith(query.replace(/\s/g, ""));
```

Comment:
Should /\s/g be a const somewhere? Looks like we're using it in more than one place. (If we do make it a const, make sure to write unit tests everywhere it's used to make sure if it get changed it doesn't break any consumers)